kms_keymaker_multiregion_replica/sns.tf_

```diff
@@ -46,8 +46,21 @@ resource "aws_sns_topic_subscription" "kms_ct_sqs" {
 
 # sets the receiver of the cloudwatch events
 # to the SNS topic
-resource "aws_cloudwatch_event_target" "sns" {
+resource "aws_cloudwatch_event_target" "decrypt" {
   rule      = aws_cloudwatch_event_rule.decrypt.name
   target_id = "${var.env_name}-sns"
   arn       = aws_sns_topic.kms_events.arn
+}
+
+#Send Cloudwatch Event to Alarm SNS Topic
+resource "aws_cloudwatch_event_target" "replicate" {
+  rule      = aws_cloudwatch_event_rule.replicate.name
+  target_id = "${var.env_name}-sns"
+  arn       = var.alarm_sns_topic_arn
+}
+
+resource "aws_cloudwatch_event_target" "update_primary_region" {
+  rule      = aws_cloudwatch_event_rule.update_primary_region.name
+  target_id = "${var.env_name}-sns"
+  arn       = var.aws_sns_topic.kms_events.arn
```

Comment:
Personally, I think it's a bit easier to leave the file as a `.tf` file and just comment these lines out (until such time that they'll either be used or removed).